kubernetes/multi-tenant/README.md

```diff
@@ -0,0 +1,6 @@
+## Script defaults
+
+* Creates a 10 MB persistent volume to load the multi tenant config file
+* Creates a 2 GB persistent volume recordings to store recordings
```

Comment:
Should this read "volume called recordings?" or just "volume to state recordings".